Makefile

```diff
@@ -0,0 +1,29 @@
+PACKAGE_NAME := copylot
+
+.PHONY: setup-develop
+setup-develop:
+	pip install -r requirements/development.txt
+	pip install -e .
+	pre-commit install
```

Comment:
I'm skeptical about requiring pre-commit for dev environments. It is not working great for everyone necessarily, I would suggest making it optional.

Reply:
Hmm okay. I edited the docs to make it clear that pre-commit is optional and added a make rule to run `black` manually if pre-commit hooks are not installed.